### ISSUE TITLE
Layout/FirstMethodArgumentLineBreak doesn't detect indentation differences on `super` method calls (only?)

### ISSUE THREAD

**Issue by pdobb** [rubocop/rubocop#9116]
When using the Layout/FirstMethodArgumentLineBreak cop, I've noticed that the first argument of a call to `super` (only?) can actually be indented at any level without problem. Further, auto-correct will pick 0 indentation, which is odd and requires manual intervention to make it look right.

--------

## Expected behavior

```ruby
# Known, bad first argument position since this is a multi-line method call
def initialize(attr1, attr2)
  super(attr1,
        attr2)
end

# Expected, required first argument position:
def initialize(attr1, attr2)
  super(
    attr1,
        attr2)
end
```

## Actual behavior

```ruby
$ rubocop --auto-correct-all --only Layout/FirstMethodArgumentLineBreak temp.rb
Inspecting 1 file
C

Offenses:

rcs/temp.rb:44:9: C: [Corrected] Layout/FirstMethodArgumentLineBreak: Add a line break before the first argument of a multi-line method argument list.
  super(attr1,
        ^^^^^

1 file inspected, 1 offense detected, 1 offense corrected

# The result of the above auto-correct is now:
def initialize(attr1, attr2)
  super(
attr1,
        attr2)

  # ...
end
```

Also Note: All of these are considered valid / no-offense:

```ruby
# Considered valid :(
def initialize(attr1, attr2)
  super(
  attr1,
        attr2)
end

# Considered valid :(
def initialize(attr1, attr2)
  super(
    attr1,
        attr2)
end

# Considered valid :(
def initialize(attr1, attr2)
  super(
                      attr1,
        attr2)
end
```

## Steps to reproduce the problem

Paste this into a ruby file and run `rubocop --auto-correct-all --only Layout/FirstMethodArgumentLineBreak temp.rb`:

```ruby
def initialize(attr1, attr2)
  super(attr1,
        attr2)
end
```

Then try adding any random number of spaces in front of `attr1` once it has been wrapped onto a newline. Note that any number of spaces is valid.

## RuboCop version

```
$ rubocop -V
1.4.2 (using Parser 2.7.2.0, rubocop-ast 1.2.0, running on ruby 2.6.6 x86_64-darwin19)
  - rubocop-performance 1.9.0
  - rubocop-rails 2.8.1
  - rubocop-rake 0.5.1
```

**Comment by Epigene** [comment]
Still getting this on default gem settings from `bundle gem`, indent for multiline `super` calls is ignored.
![Screenshot 2021-12-16 at 17 11 53](https://user-images.githubusercontent.com/2983624/146398546-b28b26ee-30cd-415c-82b5-14593ab4a5c2.png)
